src/FilterWrapper/FilterWrapper.tsx

```diff
@@ -41,6 +42,14 @@ export const FilterWrapper: React.FC<Props> = ({
 }) => {
 	const [updateBrackets, setUpdateBrackets] = useState(0);
 	const refFilterWrapper = React.createRef<HTMLDivElement>();
+	const debouncedResize = useWindowResize();
```

Comment:
Я б переименовал и переменную, и хук.
```suggestion
	const debouncedWindowSize = useDebouncedWindowSize();
```
Когда читаю суффикс "Resize" кажется что там колбэк какой-то возвращается.